tests/layer_tests/tensorflow_tests/test_tf_FusedBatchNorm.py

```diff
@@ -0,0 +1,94 @@
+        }
+        tf.compat.v1.reset_default_graph()
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            x = tf.compat.v1.placeholder(tf.float32, x_shape, 'x')
+            c_dim = x_shape[-1]
+            if data_format == "NCHW":
+                c_dim = x_shape[1]
+            scale = tf.compat.v1.placeholder(tf.float32, [c_dim], 'scale')
+            offset = tf.compat.v1.placeholder(tf.float32, [c_dim], 'offset')
+            fbn_func = fbn_dict[fbn_version]
+            if is_training:
+                mean = tf.compat.v1.placeholder(tf.float32, [c_dim], 'mean')
+                variance = tf.compat.v1.placeholder(tf.float32, [c_dim], 'variance')
+                fused_batch_norm = fbn_func(x=x, scale=scale, offset=offset, epsilon=epsilon,
+                                            mean=mean, variance=variance,
+                                            exponential_avg_factor=exponential_avg_factor, data_format=data_format,
+                                            is_training=is_training, name="FusedBatchNorm")
+            else:
+                mean = tf.compat.v1.placeholder(tf.float32, [c_dim], 'mean')
+                variance = tf.compat.v1.placeholder(tf.float32, [c_dim], 'variance')
+                fused_batch_norm = fbn_func(x=x, scale=scale, offset=offset, epsilon=epsilon,
+                                            mean=mean, variance=variance,
+                                            exponential_avg_factor=exponential_avg_factor, data_format=data_format,
+                                            is_training=is_training, name="FusedBatchNorm")
+            tf.identity(fused_batch_norm[0], name='y')
+            tf.identity(fused_batch_norm[1], name='batch_mean')
+            tf.identity(fused_batch_norm[2], name='batch_variance')
+            tf.compat.v1.global_variables_initializer()
+
+            tf_net = sess.graph_def
+
+        return tf_net, None
+
+    test_data_basic = [
+        dict(x_shape=[2, 3, 4, 5], epsilon=0.0001, exponential_avg_factor=1, data_format="NHWC", is_training=True,
+             fbn_version="v1"),
+        dict(x_shape=[2, 3, 4, 5], epsilon=0.0005, exponential_avg_factor=0.3, data_format="NHWC", is_training=True,
+             fbn_version="v2"),
+        dict(x_shape=[3, 2, 1, 5], epsilon=0.00003, exponential_avg_factor=0.7, data_format="NCHW", is_training=True,
+             fbn_version="v3"),
+        dict(x_shape=[3, 4, 2, 5], epsilon=0.0003, exponential_avg_factor=0.7, data_format="NCHW", is_training=True,
+             fbn_version="v3"),
+        # due to limitation in the layer test infrastructure - it finds tensor names for Parameter and Result nodes
+        # by get_any_name() that cannot work if some nodes fused to Parameter or Result node
+        # for inference mode, mean and variance go to outputs as batch_mean and batch_variance
+        # Now this casae will be validated with TF2 Keras BatchNormalization tests
```

Comment:
Maybe mark those tests as skip at least? Not just comment. Also does it mean that `training=false` case is not tested? Is it hard to fix tests to validate tensor `names` not `any_name`?

Reply:
Agree, I will rather to create a bug for me and mark them as xfail.

Reply:
Fixed.